libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -313,6 +313,12 @@ bytes BinaryTransform::run(Module const& _module)
 		ret += customSection(sub.first, move(data));
 		subModulePosAndSize[sub.first] = {ret.size() - length, length};
 	}
+	for (auto const& section: _module.customSections)
+	{
+		size_t length = section.second.size();
```

Comment:
This name is technically correct but I think it would be better to call it `offset` or `sectionPosition` or something like that because it better reflects how it's used here.

```suggestion
		size_t offset = section.second.size();
```

Reply:
But how is it offset?

Reply:
Sorry, it's not. I must have misread the code. See https://github.com/ethereum/solidity/pull/10219#discussion_r520036965